### DISCUSSION TITLE
Changing Backup Storage Method

### DISCUSSION THREAD

**Discussion by katekerllenevich** [pterodactyl/panel#2850] [Support] [answered]
Hey!!

I asked this in discord but I got no response, figured I would try out the new system.

Is it possible to change the backup compression type, like maybe to zstd, and how would I setup s3 backups?

Thanks!

**Comment by DomiiBunn** [comment, marked as answer]
> Is it possible to change the backup compression type, like maybe to zstd, and how would I setup s3 backups?

Yes and https://pterodox.com/guides/s3-backups.html

**Reply by katekerllenevich** [reply]
Thanks!

**Reply by DaneEveritt** [reply]
@DomiiBunn thanks for the write-up, but would you mind also posting the contents here. Would like to avoid this turning into a 6 months to a year down the line the link doesn't work and people don't know how to find the answers.

**Reply by DomiiBunn** [reply]
I'll try to keep it alive for longer :) also didn't mention but if you want to coppy any of these into official be my guest
